tests/Unit/Service/Sync/SyncServiceTest.php

```diff
@@ -25,6 +27,13 @@
 use Psr\Log\LoggerInterface;
 
 class SyncServiceTest extends TestCase {
+
+	/** @var IMAPClientFactory */
+	private $clientFactory;
+
+	/** @var Horde_Imap_Client_Socket */
+	private $client;
```

Comment:
```suggestion
	private IMAPClientFactory&MockObject $clientFactory;
	private Horde_Imap_Client_Socket&MockObject $client;
```

and import `use PHPUnit\Framework\MockObject\MockObject;`